scripts/release-prod

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+read -p "Releasing master(CI) to QA environment. Do you wish to continue? (Y/n) " decision
```

Comment:
Shouldn't this be "Releasing qa to prod environment"? Or something like it?

Reply:
Good catch. Updated.